CONTRIBUTING.md

```diff
@@ -40,11 +46,10 @@ AUTH0_CLIENT_ID=<your-auth0-client-id>
 AUTH0_CLIENT_SECRET=<your-auth0-client-secret>
 ```
 
-Then, run `make testacc`. 
+Then, run `docker-compose up -d && make testacc`. 
```

Comment:
Thoughts about bundling the `docker-compose up` directly into a separate command in the makefile? Perhaps `make testacc_local`? The makefile does a good job of hiding away a lot of the implementation details and we don't want to require folks to need to know to run the docker compose locally.

However, this would just be for local invocation, I do like that it's separated in the Github workflow above.

Reply:
Further, you're omitting the `--build` arg here whereas it exists in the workflow, I forget if building the image is a default behavior of `up` but would think that we'd want to be consistent in both places.

Reply:
Sure, we can add a makefile command to bootstrap the dev env. 

When `docker-compose up` is ran and the image is not available on the host machine it will always default to building it. The `--build` is there just to always force rebuilding and not use the one from cache.